### PR TITLE
Implement hashing with murmur

### PR DESCRIPTION
**Pull request by alanchristensen**
Java client PR https://github.com/Unleash/unleash-client-java/pull/45

### CODE REVIEW

**Comment by ivarconr** [comment]
thanks! :+1:

File: package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "eslint-config-finn-prettier": "^3.0.0",
     "in-publish": "^2.0.0",
     "mkdirp": "^0.5.1",
+    "murmurhash3js": "^3.0.1",
```
Comment (ivarconr) [inline package.json]: minor: `murmurhash3js` nees to be a dependency and not a devDependecy.

Comment (ivarconr) [reply package.json]: fixed via https://github.com/Unleash/unleash-client-node/commit/e56d9c5d59ac37983ae8d410fb86338bd7efdcb9

Comment (alanchristensen) [reply package.json]: Whoops, thanks for the correction!

**Pull request merged**